frontend/public/locales/fr/translation.json

```diff
@@ -604,7 +604,7 @@
   "visualizeDataBox": {
     "visualizeTheData": "Visualisez les données",
     "youCanQuicklyExploreEtc": "Vous pouvez rapidement explorer notre base de données publique en utilisant notre application <1>Tournesol Data Visualization</1> réalisée avec Streamlit.",
-    "personCorrelationCoefficientMatrix": "Matrice des coefficient de corrélation de Pearson entre les scores des critères de comparaison (2022/10/10)."
+    "personCorrelationCoefficientMatrix": "Matrice des coefficient de corrélation de Pearson entre les scores des critères de comparaison (2022/06/13)."
```

Comment:
Oops, the year is still 2022 here.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1611/commits/fd5a35d0896b1f3b82db06dfdb20c287faf64f26